onnx/test/checker_test.py

```diff
@@ -91,15 +91,23 @@ def test_check_function_nested(self) -> None:
             func_nested_identity_add_nodes,
             func_nested_opset_imports,
         )
-        checker.check_function(func_nested_identity_add)
+        ctx = checker.C.CheckerContext()
+        ctx.ir_version = 3
+        ctx.opset_imports = {"": onnx.defs.onnx_opset_version()}
```

Comment:
Maybe better to add a specific opset version, instead of `onnx_opset_version`. The function itself uses opset version 14. Is this trying to test with a different opset version for model or same? In either case, better to make it a specific version.

Reply:
Thanks for catching this! I updated this to `ctx.opset_imports = {"": 14}`. Should be the same version.

Reply:
The DEFAULT is problematic because it keeps changing with every release. I think it is better that the tests are explicit about what they are testing: they can be parametrized if we wish to test against multiple opset versions, or they can target a specific version. (A better fix would be to drop the DEFAULT, but that could cause backward compat issues for other users, so that's not worth doing for now, though worth discussing for the future.)

Reply:
Thanks for your defaulted explanation. I think parameterized tests would be more efficient and `checker_test.py` should use `parameterized` like some of the other test files. I will work on updating this. 

For this PR I will be using opset version 14 explicitly.